### DISCUSSION TITLE
dependabot support for reusable workflows

### DISCUSSION THREAD

**Discussion by brianmay** [community/community#8088] [Code Security] [answered]
It would be really good if dependabot could automatically detect outdated workflows and update them.

Unless I missed something it looks like the `package-ecosystem: github-actions` setting in `.github/dependabot.yml` does not do this.

**Comment by pivovarit** [comment]
The workaround is to reference the latest version (like `@main`), but I'd love for Dependabot to be able to identify new tags.

**Reply by flaxel** [reply]
Yes, it's a workaround, but the disadvantage of this is that you don't really notice new features or changes to the actions, which can break workflows. So I would also love to see that Dependabot recognize the reusable workflows.

**Reply by brianmay** [reply]
Agreed. A single innocent change to a workflow could break every single project in one go.

**Comment by cytopia** [comment]
Is there any update on when we might expect this feature, if at all?

**Comment by AidarGatin** [comment]
Hi Guys, actually this feature already supported. What you need is to grant Dependabot access to your private with reusable workflows. And of course you should have Release and Tag in place.  

https://github.blog/changelog/2023-03-13-dependabot-updates-support-reusable-workflows-for-github-actions/

<img width="1346" alt="Screenshot 2023-05-10 at 12 14 38 PM" src="https://github.com/community/community/assets/2663867/c94c031c-c526-430f-847a-c3f95eaa5c1b">

**Reply by brianmay** [reply]
Yes, I believe it has been implemented since I wrote this. Haven't had a chance to test it yet.

**Reply by wenovus** [reply, marked as answer]
Yep this is possible: https://github.com/openconfig/goyang/pull/254

**Comment by Nikol1988** [comment]
[](https://github.com/orgs/community/discussions/8088)
